src/playground/sample_data_ingestion/jobs/refresh_job.py

```diff
@@ -0,0 +1,145 @@
+# opensearch-py==2.0.0
+from opensearchpy import OpenSearch
```

Comment:
Can you add README.md for configuration files and usage as well? Thanks

Reply:
Done, `README.md` was added.